### DISCUSSION TITLE
PWA or web version?

### DISCUSSION THREAD

**Discussion by tomyo** [anyproto/contributors#141] [Q&A] [answered]
I want to be able to use anytype on a browser, is there any plans for that?

Thank you.

**Comment by fuksman** [comment, marked as answer]
Hi @tomyo,

We have received [requests](https://community.anytype.io/t/web-app-web-client/1381/19) from users, but this feature is not on our roadmap yet. The use of browsers can potentially weaken the security model of Anytype.

**Reply by tomyo** [reply]
Thanks for the reply.

> The use of browsers can potentially weaken the security model of Anytype.

This potentially sounds like a vague excuse to me.

Thanks again.

**Comment by LYUYork** [comment]
how is this going? add this feature to the roadmap?

**Comment by alensiljak** [comment]
Perhaps it can be done with WASM, one day.

**Comment by KarstenDE** [comment]
Hi all,
I would highly appreciate this solution in general. It offers some use cases especially using Anytype e.g on business devices where the installation of the app is restricted.
Looking forward to :-)

**Reply by KarstenDE** [reply]
Sure, this can be limited to self hosting environments to provide the required components.
